bugsnag-android-performance/detekt-baseline.xml

```diff
@@ -26,6 +26,7 @@
     <ID>MagicNumber:SpanKind.kt$SpanKind.CLIENT$3</ID>
     <ID>MagicNumber:SpanKind.kt$SpanKind.CONSUMER$5</ID>
     <ID>MagicNumber:SpanKind.kt$SpanKind.PRODUCER$4</ID>
+    <ID>NewLineAtEndOfFile:SpanEndCallback.kt$com.bugsnag.android.performance.SpanEndCallback.kt</ID>
```

Comment:
This should not be part of the baseline. `SpanEndCallback.kt` should have a newline at the end of the file